rke/k8s_rke_system_images.go

```diff
@@ -5778,6 +5778,50 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/kubelet-pause:v0.1.6",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.15.13",
 		},
+		// Enabled out of band in v2.5.11
```

Comment:
small change, we should say `Enabled out of band post v2.5.11` because we're releasing this version after v2.5.11 is tagged and released https://github.com/rancher/rancher/releases/tag/v2.5.11.

Reply:
Sure. pushing the changes.